SECP-service/src/main/resources/assets/app/css/login.css

```diff
@@ -0,0 +1,88 @@
+@import url("https://fonts.googleapis.com/css?family=Raleway:400,700");
```

Comment:
I can see duplication of css here. You should probably use the @import to bring the css from register here.